pkgs/development/interpreters/guile/3.0.nix

```diff
@@ -126,6 +130,20 @@ builder rec {
 
   setupHook = ./setup-hook-3.0.sh;
 
+  passthru = {
+    updateScript = writeScript "update-guile-3" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p curl pcre common-updater-scripts
+
+      set -eu -o pipefail
+
+      # Expect the text in format of '"https://ftp.gnu.org/gnu/guile/guile-3.0.8.tar.gz"'
+      new_version="$(curl -s https://www.gnu.org/software/guile/download/ |
+          pcregrep -o1 '"https://ftp.gnu.org/gnu/guile/guile-(3[.0-9]+).tar.gz"')"
+      update-source-version ${pname} "$new_version"
```

Comment:
We have `guile = guile_2_2;` in all-packages.nix, will it be possible that this will bump guile_2_2 instead? See sample run in #164197. Sorry I am really not expert of updateScript.

Reply:
Oh, that's unexpected. The file location looks off. Looking.

Reply:
Proposed the fix as https://github.com/NixOS/nixpkgs/pull/164512. All my fault of having local override of `guile = guile_3_0`.